### PR TITLE
Adds functionality with airtable to track files

### LINKED ISSUE #63: Swap link for attendance form
The link for the attendance form was the edit link by accident.

Can we swap it out for this one? https://forms.gle/zXg75YumTuABXSs19

P

### LINKED ISSUE #73: Add ability to share links via Airtable
Proposed interaction:

```
=> @scout fetch doc123

=> @scout track doc123 (link) # starts tracking a document by updating Airtable with a provided link

=> @scout untrack doc123 
```

### PR DESCRIPTION
**Pull request by ZadenRB**
Closes #63 
Closes #73

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ Brigades:
     CodeOfConductMessageID: ENC[AES256_GCM,data:K3UDjApDuHTlupow8ci2J+JP,iv:iuqIFVSuCb0itgDvk0feap3owHLKMNgfCgBacPPZRnk=,tag:kkNIDPP0n4yaGqlBjOVisA==,type:str]
     AgendaFolderID: ENC[AES256_GCM,data:M2uw6toYkWJAMkkp9bF/gFZ/eNlPcwX9S+aZ5Jj5/MTm,iv:tTdB0/bPYXHGR+8aBBStqEKVvy7r3OLid7EYtcfSgxc=,tag:MJEcC4pLYrrA/lwiH17Oug==,type:str]
     LocationString: ENC[AES256_GCM,data:YKzQOTpt0Vk7hoGA97Q=,iv:YPGlGLXDhOWF/uKp/vJ1MmZSvzCTd0qc5TIMRNpT5gA=,tag:7WcMnW77CrIWTX/CXoQgrg==,type:str]
+    AirtableBaseID: ENC[AES256_GCM,data:MQO7jYyY/w0hFbFfdYa4HBs=,iv:MdLoh0Suvcxr9Bg767TusA2di+ybbV6XuCulXRDdzyo=,tag:yRDZ+mr82bHzEAp269crEQ==,type:str]
```
Comment (dviramontes) [inline config.yaml]: 🔒 👍

File: pkg/discord/main.go [hunk 2]
```diff
@@ -535,6 +544,122 @@ func sendGithubUsername(data CommandData) {
...
+			fmt.Println("error sending message to channel,", err)
+		}
+		return
+	} else if err != nil {
+		if _, err := data.Session.ChannelMessageSend(data.ChannelID, "Failed to check if a file with the name **"+fileName+"** is already tracked."); err != nil {
+			fmt.Println("error sending message to channel,", err)
+		}
+		return
+	}
+	files := client.Table("Files")
```
Comment (dviramontes) [inline pkg/discord/main.go]: does this create a table every time ?

Comment (ZadenRB) [reply pkg/discord/main.go]: No, just fetches the table named "files". Would be inline(`client.Table("Files").List(...)`) but the airtable library takes the address of the table, so it has to be set to a variable first.

Comment (dviramontes) [reply pkg/discord/main.go]: gotcha 👌

**Pull request merged**